runtime/onert/core/src/util/ConfigSource.cc

```diff
@@ -30,8 +30,10 @@ namespace util
 {
 
 static std::unique_ptr<IConfigSource> _source;
+static std::unique_ptr<IConfigSource> _source_ext;
 
 void config_source(std::unique_ptr<IConfigSource> &&source) { _source = std::move(source); }
+void config_source_ext(std::unique_ptr<IConfigSource> &&source) { _source_ext = std::move(source); }
```

Comment:
I'm not sure existing `config_source()` is used from some external projects.
If `config_source()` is not used then we can change this
```cxx
void config_source(std::unique_ptr<IConfigSource> &&source) { _source_ext = std::move(source);
```
and NOT add new `config_source_ext()`